python/lsst/afw/detection/_psf.cc

```diff
@@ -52,10 +55,12 @@ void wrapPsf(utils::python::WrapperCollection& wrappers) {
     wrappers.addSignatureDependency("lsst.afw.fits");
 
     auto clsPsf = wrappers.wrapType(
-            py::class_<Psf, std::shared_ptr<Psf>, typehandling::Storable, PyPsf<>>(wrappers.module, "Psf"),
+            py::class_<Psf, PySharedPtr<Psf>, typehandling::Storable, PsfTrampoline<>>(
+                wrappers.module, "Psf"
+            ),
             [](auto& mod, auto& cls) {
                 table::io::python::addPersistableMethods<Psf>(cls);
-                cls.def(py::init<>());  // Dummy constructor for pure-Python subclasses
+                cls.def(py::init<bool, size_t>(), "init", "isFixed"_a=false, "capacity"_a=100);  // Constructor for pure-Python subclasses
```

Comment:
Is the "init" here correct? I don't think I've ever seen a string argument used with `py::init`.

Reply:
It's a docstring for `__init__`.  Pretty uninformative here though :) ; I'll delete it.